src/Servers/Connections.Abstractions/src/Microsoft.AspNetCore.Connections.Abstractions.csproj

```diff
@@ -13,7 +13,6 @@
 
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore.Http.Features" />
-    <Reference Include="System.IO.Pipelines" />
```

Comment:
Nice cleanup. This was brought in transitively through Microsoft.AspNetCore.Http.Features since forever